### ISSUE TITLE
Description of Model

### ISSUE THREAD

**Issue by xcmyz** [ivanvovk/DurIAN#2]
Wonderful work!
I have one question when I look into your readme.
The difference of durian between fastspeech is its autoregressive structure, which lets durian can generate better wav. It use previous output as input of current step. However, in the picture of baseline model, it is different from description of paper.
![image](https://user-images.githubusercontent.com/39415743/84572709-7a015f80-adce-11ea-94b5-2c0abe20e15f.png)
Besides, I think it is important to generate r steps of mel spectrogram at once (refer paper of tacotron), which is also difficulty when implementing durian. I hope you can implement this part.

Thank you!

**Comment by ivanvovk** [comment]
Yeah, you're right. My baseline model doesn't take into account any info about previous steps. So, that's why it is baseline :) However, for paper-based version, which is also implemented here and also reported in README, self-conditioning decoder is provided - I wrote it in key differences for "vanilla version". I've done baseline just for testing. I guess I just need to remove it to not make others got confused. What about several steps generation at one time - yes, it is a good idea and I think it can be easily added to current implementation in the near future, I will mark it.
